src/app.js

```diff
@@ -55,12 +55,12 @@ class Messenger extends EventEmitter {
   /*:: options: Object */
   /*:: pages: Object */
   constructor({
-      hookPath = '/webhook',
```

Comment:
They changed the indention rules in Eslint v4 http://eslint.org/docs/user-guide/migrating-to-4.0.0

I don't really like them, but if that's what JavaScript is doing 🤷‍♂️ I'll live with it